cloud/ec2.go

```diff
@@ -775,6 +786,15 @@ func (m *ec2Manager) GetInstanceStatus(ctx context.Context, h *host.Host) (Cloud
 
 	instance, err := m.client.GetInstanceInfo(ctx, id)
 	if err != nil {
+		// terminate an unknown host in the db
+		if err == noReservationError {
+			grip.Error(message.WrapError(h.Terminate(evergreen.User, "host is unknown to AWS"), message.Fields{
```

Comment:
I'd suggest a slightly different message here so that we can distinguish this case from the GetInstanceStatuses case.

Reply:
done